dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,24 +1,16 @@
 module SurveyResultsHelper
   def show_survey?
-    # rubocop:disable Lint/UnreachableCode
-
-    # Disabled for all users on 2016-03-30 after being shown for thirty days.
-    return false
-
     # Reasons we would not show the survey.
     unless current_user
       return false
     end
-    unless current_user.teacher?
```

Comment:
I meant revive. Doubtful we'll ever have two at the same time (at least in the near future). Just brainstorming if we start adding more than these 2 surveys and they all have different show conditions how to keep it easily separated. Obviously this one is easy to add back in when we need it so not blocking.